OkienkoStartowe.java

```diff
@@ -0,0 +1,280 @@
+
+                }
+                else
+                {
+                    biletNie.setVisible(false);
+                    biletTak.setVisible(true);
+
+                }
+            }
+        });
+
+        wJednaStrone.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+                dataPrzylotu.setDisable(true);
+                System.out.println("W jedna strone");
+            }
+        });
+
+        wObieStrony.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+                dataPrzylotu.setDisable(false);
+                dataWylotu.setDisable(false);
+                System.out.println("W obie strony");
+
+            }
+        });
+
+
+        String [] lista = {"Krakow", "Warszawa", "Sztokholm", "Wieden", "Praga", "Amsterdam", "Glasgow", "Istanbul"};
+        ChoiceBox<String> miastoWylot = new ChoiceBox<>();
+        miastoWylot.setLayoutX(120);
+        miastoWylot.setLayoutY(150);
+        miastoWylot.getItems().addAll(lista);
+        miastoWylot.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+
+                System.out.println("Wylot do " + miastoWylot.getValue() );
+            }
+        });
+
+        String [] lista2 = {"Kopenhaga", "Paryz", "Londyn", "Porto", "Berlin", "Barcelona", "Poznan", "Ateny"};
+        ChoiceBox<String> miastoPrzylot = new ChoiceBox<>();
+        miastoPrzylot.setLayoutX(300);
+        miastoPrzylot.setLayoutY(150);
+        miastoPrzylot.getItems().addAll(lista2);
+        miastoPrzylot.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+
+                System.out.println("Przylot z " + miastoPrzylot.getValue() );
+            }
+        });
+
+        Label lotZ = new Label("Z");
+        lotZ.setLayoutX(160);
+        lotZ.setLayoutY(120);
+
+        Label lotDO = new Label("DO");
+        lotDO.setLayoutX(340);
+        lotDO.setLayoutY(120);
+
+        Label data = new Label("DATA  ");
+        data.setLayoutX(120);
+        data.setLayoutY(230);
+        data.setTextFill(Color.GREEN);
+
+        Label pasazer = new Label("LICZBA BILETOW  ");
+        pasazer.setLayoutX(300);
+        pasazer.setLayoutY(230);
+        pasazer.setTextFill(Color.GREEN);
+
+
+        Button zaplac = new Button("Zaplac teraz");
+        zaplac.setLayoutX(600);
+        zaplac.setLayoutY(500);
+        zaplac.setTextFill(Color.ORANGE);
+        zaplac.setVisible(false);
+        zaplac.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+                System.out.println("Płacimy");
+            }
+        });
+
+        // Cena podawana z Random, nie mam pomysłu jak zrobic wynik   (Cena * liczbaBiletow )....

```

Comment:
Dodac metode ktora oblicza cene ` private String getAllPriceForTickets(
      int randomPriceForOneTicket, ChoiceBox<Integer> choiceBoxWithNumberOfTickets) {
    int numberOfTickets = 0;
    if (choiceBoxWithNumberOfTickets.getValue() != null) {
      numberOfTickets = choiceBoxWithNumberOfTickets.getValue();
    }
    return "CENA ZA BILETY WYNOSI :  " + randomPriceForOneTicket * numberOfTickets + " PLN";
  }`
Poprzez ta metode zmieni sie to wywolanie przy Label cena i bedzie tak wygladac:
`Label cena = new Label(getAllPriceForTickets(Cena, liczbaBiletow));
    cena.setLayoutX(320);
    cena.setLayoutY(430);
    cena.setTextFill(Color.RED);
    cena.setVisible(false);
    szukaj.setOnAction(
        event -> {
          System.out.println("Szukamy");
          cena.setText(getAllPriceForTickets(Cena, liczbaBiletow));
          zaplac.setVisible(true);
          cena.setVisible(true);
        });`